test/e2e/llmisvc/diagnostic.py

```
@@ -86,18 +97,99 @@ def kinds_matching_by_labels(namespace: str, labels, skip_api_kinds=None):
+        label = "previous" if previous else "current"
+        try:
+            logs = core.read_namespaced_pod_log(
+                name=pod_name,
+                namespace=namespace,
+                container=container_name,
+                previous=previous,
+                tail_lines=200,
+            )
+            log("# -- logs (%s) --", label)
+            log("%s", logs or "(empty)")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)